webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala

```diff
@@ -211,12 +226,17 @@ class VolumeTracingService @Inject()(
         Fox.serialCombined(additionalCoordinateList)(additionalCoordinates => {
           val mag = vec3IntFromProto(resolution)
           for {
+            fallbackLayer <- getFallbackLayer(tracingId)
             bucketPositionsRaw <- volumeSegmentIndexService.getSegmentToBucketIndexWithEmptyFallbackWithoutBuffer(
+              fallbackLayer,
               tracingId,
               a.id,
               mag,
+              None, //TODO: ???
```

Comment:
yes, None means the latest version here, which is the only one that this operation makes sense for